package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Familia",
+  "version": "1.0.0",
+  "description": "An app to track family's required groceries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start":"nodemon src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FACK1/Familia.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Where are your dependencies/devDependencies? You need to include express and express-handlebars so people who clone your project don't have to do this manually.